pkg/skaffold/runner/runcontext/context.go

```diff
@@ -141,6 +141,14 @@ func (rc *RunContext) Deployers() []latest.DeployType { return rc.Pipelines.Depl
 
 func (rc *RunContext) TestCases() []*latest.TestCase { return rc.Pipelines.TestCases() }
 
+func (rc *RunContext) StatusCheck() bool {
```

Comment:
should this return `*bool` since you want to handle the `nil` case against the `skaffold.yaml` property elsewhere?